network/config/src/node_config.rs

```diff
@@ -0,0 +1,16 @@
+
+
+struct NetworkNodeConfig {
+     signing_public_key: Ed25519PublicKey,
+     network_identity_public_key: x25519::PublicKey,
+     network_address: RawNetworkAddress,
+}
+
+struct NetworkMembershipConfig {
+    network_id : string,
+    nodes: Map<AccountdAddress, NetworkNodeConfig>,
+}
+
+struct NetworkOperationalConfig {
+
+}
```

Comment:
File to be removed.  Part of scratch work.

Reply:
Yeah, let's not do this work in this PR.